index.js

```diff
@@ -213,11 +214,11 @@ module.exports = {
     'spaced-comment': 0,
     'wrap-regex': 0,
     'arrow-body-style': 0,
-    'arrow-parens': 0,
+    'arrow-parens': 2,
     'arrow-spacing': 0,
     'constructor-super': 2,
     'generator-star-spacing': 0,
-    'no-confusing-arrow': 2,
+    'no-confusing-arrow': 0,
```

Comment:
👍 

https://eslint.org/docs/rules/no-confusing-arrow#disallow-arrow-functions-where-they-could-be-confused-with-comparisons-no-confusing-arrow

Reply:
👍